src/selectors/getChildren.ts

```diff
@@ -185,6 +187,10 @@ export const getAllChildrenSorted = (state: State, id: ThoughtId): Thought[] =>
     return getChildrenSortedCreated(state, id)
   } else if (sortPreference.type === 'Updated') {
     return getChildrenSortedUpdated(state, id)
+  } else if (sortPreference.type === 'Note') {
+    return getChildrenRanked(state, id).sort(
+      sortPreference.direction === 'Desc' ? compareThoughtByNoteDescending(state) : compareThoughtByNote(state),
+    )
```

Comment:
I'm not sure why this breaks from the pattern and does something different with the other sort types.